test/compose/cdi_device/setup.sh

```diff
@@ -0,0 +1,9 @@
+if is_rootless; then
+    reason=" - can't write to /etc/cdi"
+    _show_ok skip "$testname # skip$reason"
+    exit 0
+fi
+
+mkdir -p /etc/cdi
+mount -t tmpfs tmpfs /etc/cdi
```

Comment:
That can also effect the host, if other things try to use cdi in the meantime.
I guess its fine as almost nobody runs these tests, I am just not a fan of these global changes.

But I guess having a test is better than not having it so that is fine.

Reply:
yeah, we need https://github.com/containers/podman/pull/21448 to fix both CDI tests